client_restaurant/shared/src/commonMain/kotlin/presentation/main/MainScreenModel.kt

```diff
@@ -25,46 +25,42 @@ class MainScreenModel(private val restaurantId: String) : BaseScreenModel<MainSc
     }
 
     private suspend fun callee(): List<Restaurant> {
-        return getOwnerRestaurantsInformationUseCase("")
+        return getRestaurantsInformation("")
```

Comment:
I think that this function should not take any parameters, owner ID should be retrieved from Local, not not viewModel responsibility what do you think?